lib/beaker/host/windows/exec.rb

```diff
@@ -108,4 +108,13 @@ def selinux_enabled?()
     false
   end
 
+  # Determin if cygwin is actually installed on the SUT.
+  #
+  # @return [Boolean]
+  def cygwin_installed?
+    output = exec(Beaker::Command.new('cygcheck -c cygwin'), :accept_all_exit_codes => true).stdout 
+    return true if output.match(/cygwin/) and output.match(/OK/)
```

Comment:
This checks the content of the response because cygcheck still returns a `0` if the package is not found.

Beaker does capture the error level, and you could ostensibly say that by virtue of running `cygcheck` you can confidently say that cygwin itself is installed; I opted for this more verbose method to ensure that the cygwin installation was installed without errors.